packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -178,3 +180,43 @@ describe("segment graph", () => {
         });
     });
 });
+
+describe("locked layer", () => {
+    const apiOptions = {flags: {mafs: {segment: true}}};
+    it("should render locked points", async () => {
+        // Arrange
+        const {container} = renderQuestion(
+            segmentWithLockedPointsQuestion,
+            apiOptions,
+        );
+
+        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+        const points = container.querySelectorAll(
+            // Filter out the interactive points' circles
+            "circle:not(.mafs-movable-point):not(.mafs-movable-point-hitbox):not(.mafs-movable-point-focus):not(.mafs-movable-point-ring):not(.mafs-movable-point-point)",
+        );
```

Comment:
This is absolutely not ideal. Would love to pair with someone if they know a better to way to find these circles.

Reply:
Would this work: `"circle:not([class*='mafs-movable-point'])"`

Reply:
@mark-fitzgerald YES!